dev/implementations/storage_impl_definitions.h

```diff
@@ -154,7 +124,8 @@ namespace sqlite_orm {
         }
 
         template<class H, class... Ts>
-        sync_schema_result storage_impl<H, Ts...>::schema_status(sqlite3* db, bool preserve) const {
+        sync_schema_result
+        storage_impl<H, Ts...>::schema_status(sqlite3* db, bool preserve, std::vector<table_info>& dbTableInfo) const {

```

Comment:
why do you pass non const reference? better make it `const` cause `dbTableInfo` is never modified

Reply:
schema_status() changes parameter dbTableInfo in line 19496, in the call to `if(this->calculate_remove_add_columns(columnsToAdd, storageTableInfo, dbTableInfo)) {`
otherwise I would have passed `const reference`

Reply:
please see `dbTableInfo.erase(dbColumnInfoIt);` inside `bool storage_impl_base::calculate_remove_add_columns`

Reply:
oh I see. Sorry. Everything is ok